docs/configuration.asciidoc

```diff
@@ -218,3 +249,34 @@ This table outlines the attributes and their corresponding permissions:
 The configurable parameters provided by the Elastic APM agent aim to help configuring common use cases in an easy way, in most of the cases it means to act as a facade between your application and the OpenTelemetry Java SDK that this agent is built on top.
 If your project requires to configure more advanced aspects of the overall APM processes, you could directly apply that configuration using the https://opentelemetry.io/docs/instrumentation/java/getting-started/[OpenTelemetry SDK], which becomes available for you to use within your project by adding the Elastic agent plugin, as explained in <<setup,the agent setup guide>>.
 Said configuration will be used by the Elastic agent for the https://opentelemetry.io/docs/concepts/signals/[signals] it sends out of the box.
+
+[float]
+[[configuration-dynamic]]
+=== Dynamic configuration image:./images/dynamic-config.svg[]
+
+Configuration options marked with Dynamic true can be changed at runtime when set from Kibana's {kibana-ref}/agent-configuration.html[central configuration].
+
+[float]
+=== Option reference
+
+This is a list of all configuration options.
+Click on a key to get more information.
```

Comment:
Is this supposed to be here?

Reply:
I'm not sure, seems like in the Java Agent docs the options become clickable. So it was mostly because it's like this over there, but based on what you mentioned here it seems it's not really needed so I'm fine with removing it.